pages/single-form.jsx

```diff
@@ -91,6 +91,7 @@ var SingleForm = React.createClass({
         </div>
         <SmallPrint stripeNotice={true} />
         <Footer/>
+        <script async src="https://checkout.stripe.com/checkout.js"></script>
```

Comment:
Well, the amount is likely prefilled via snippet.

So it's privacy then donate.